# Defines the discretized states of Hyq with joint angles

HyqStateJoints[] states      # trajectory of states